server/views/layout.html

```diff
@@ -26,9 +26,9 @@
   <meta property="keywords" content="{{metaKeywords}}" />
   <meta property="description" content="{{metaDescription}}" />
   <meta name="google-site-verification" content="HjuKUejT64a0rBdUxxtbnIKNB4_cZ3X3hlXOxk4_OX8" />
-  {% if metaNoIndex %}
+  {#% if metaNoIndex %#}
```

Comment:
i've not noticed this if statement before, but was it taken out due to a need on the ticket? I think in /robots.txt we block all indexing by default on the service anyway

Reply:
This was a request from @danleech-defra . As I understand it the noindex setting was only being applied to the the  river-and-sea-levels and alerts-and-warnings list pages in this test. The removal was to ensure whilst in beta it was applied to all pages. The aim was to reintroduce when live hence commenting out as opposed to deleting. Agreed, the current robots.tx tsetting blocks all by default.  Therefore whilst that is in place this is just belt and braces. Can remove if wished.